project_event/models/project_task.py

```diff
@@ -102,6 +102,9 @@ class Task(models.Model):
         default='draft',
         track_visibility='onchange',
     )
+    reservation_event_id = fields.Integer(
+        string='Reservation event',
+    )
 
     @api.onchange('resource_type')
```

Comment:
@eilst : Whatis the purpose of setting to "None" on onchange of resource type. Not clear with this method usability

Reply:
To oly have one kind of resource